typescript/helloworld/src/app/app.ts

```diff
@@ -21,8 +21,9 @@ export class HelloWorldApp extends RouterApp<HelloWorldFactories> {
     public readonly core: HyperlaneCore,
     contractsMap: HyperlaneContractsMap<HelloWorldFactories>,
     multiProvider: MultiProvider,
+    foreignDeployments: ChainMap<Address> = {},
   ) {
-    super(contractsMap, multiProvider);
+    super(contractsMap, multiProvider, undefined, foreignDeployments);
```

Comment:
I think you don't need any changes -- the reason we still need this HelloWorldApp is because it's used for the deployer / checker. The foreign deployment stuff is just so that we only try to interact with the non-foreign EVM deployments, and that the foreign deployments are still enrolled as remote routers on the EVM contracts